testing/vault/vault.initd

```diff
@@ -10,9 +10,10 @@ pidfile="/run/${SVCNAME}.pid"
 
 start_pre() {
 	checkpath -f -m 0644 -o ${SVCNAME}:${SVCNAME} "$VAULT_LOG_FILE"
+	setcap cap_ipc_lock=+ep $(readlink -f $(which vault))
```

Comment:
I understand why it is used and I didn’t mean that you should not use `setcap` at all, just that it should be set in APKBUILD, not post-install script. abuild and apk handles these attributes (if not for you, then use the latest abuild and apk versions).